client/src/modules/multiple_payroll_indice/modals/config.modal.js

```diff
@@ -77,15 +87,36 @@ function ConfigIndicePaiementModalController(
         vm.employee.rubrics.forEach(r => {
           vm.selectedRubrics[r.rubric_id] = r.rubric_value;
         });
+
         vm.rubrics = result.rubrics;
+
         vm.rubrics.forEach(r => {
           rubricsMap[r.id] = r;
           if (!vm.selectedRubrics[r.id]) {
             vm.selectedRubrics[r.id] = r.value || 0;
           }
         });
+
+        return StaffingIndice.getRubricsGradeEmp({ employee_uuid : params.uuid });
+      })
+      .then(data => {
+        vm.rubricsGrade = data;
+
+        vm.rubrics.forEach(rb => {
+          vm.rubricsGrade.forEach(rub => {
```

Comment:
Couldn't you just find the rubricsGrade using "find()" rather than looping through to find it?

Also, the choice of temporary variables is confusing.  "rb" is fine for rubric.  But "rub" should be something to tie it to rubricsGrade.  For instance, "rg" or "rubGrade" would be clearer; if you do this, you may want to change "rb" to "rub".